runtimes/eden/src/version.rs

```diff
@@ -48,7 +48,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	// do the same thing.
 	// Non-consensus-breaking optimizations are about the only changes that could be made which
 	// would result in only the `impl_version` changing.
-	impl_version: 0,
+	impl_version: 1,
```

Comment:
The advice for substituting code was to keep all the versions the same (disclaimer: I haven't investigated the advice independently). So we need to make sure this is ok.

Reply:
Lets just roll back this as it is an increased risk for no major benefit